shed/event_streams.py

```diff
@@ -256,7 +257,9 @@ def curate_streams(self, nds):
         else:
             names, docs = nds
             name = names
-            docs = (docs,)
+            # transform {} to ({}, ) if needed
+            if not isinstance(nds[1], tuple):
+                docs = (docs,)
```

Comment:
I would add this after
```python
        # this is needed in case some elements of docs were a list
        # here we assume doc is always a dict and the list of docs a tuple
        newdocs = list()
        for doc in docs:
            # for case of ((name, ({}, {})), (name, ({}, {})))
            if isinstance(doc, tuple):
                newdocs.extend(doc)
            else:
                newdocs.append(doc)

        docs = tuple(newdocs)
```
so that `s.curate_streams((('start', ({}, {})), ('start', {})))`
gives `('start', ({}, {}, {}))`
and not
`('start', (({}, {}), {})))`